src/kernel/x64/cmos-time-x64.cc

```diff
@@ -0,0 +1,112 @@
+// Copyright 2015 Runtime.JS project authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Using code adapted from http://wiki.osdev.org/CMOS#Reading_All_RTC_Time_and_Date_Registers
+// OSDev disclaimer: http://wiki.osdev.org/OSDev_Wiki:General_disclaimer
+
+#include "cmos-time-x64.h"
+
+namespace rt {
+
+CMOSTime::CMOSTime() {
+  second = 0;
+  minute = 0;
+  hour = 0;
+  day = 0;
+  month = 0;
+  year = 0;
+}
```

Comment:
pretty much the same, but you can use initializer list instead like
```c++
CMOSTime::CMOSTime()
  : second(0),
    minute(0),
    ...
    year(0)  {}
```

btw, private variables should probably be named with underscore like `second_`